src/Dialog/Confirmation/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import { Box } from '@mui/system';
+
+export type ConfirmationDialogProps = {
+  title: string;
+  content: string;
+  onClose: () => void;
+  onConfirm?: () => void;
+};
+
+const ConfirmationDialog: React.FC<ConfirmationDialogProps> = ({
+  title,
+  content,
+  onClose,
+}) => {
+  const handleConfirm = () => {
+    onClose();
+  };
+
+  return (
+    <Box>
```

Comment:
Dont redeclare the full thing here - move this into `Dialog.tsx`. I feel like this and that got mixed up in terms of which is the base and which is the implementation.